<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillEdit</class>
 <widget class="QDialog" name="SkillEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fID"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Level</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fLevel"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Passive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fPassive">
     <item>
      <property name="text">
       <string>False</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>True</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Target</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fTarget">
     <item>
      <property name="text">
       <string>Single neighbour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Area range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Area neighbour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Self</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single range projectile</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Caster area</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Range</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fRange"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="fRadius"/>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="fDescription"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
